ImperatorToCK3/Source/Imperator/Provinces/Province.h

```diff
@@ -32,6 +33,7 @@ namespace ImperatorWorld
 		std::string religion;
 		int owner = 0;
 		int controller = 0;
+		unsigned int buildingsCount = 0;
```

Comment:
why, do you intend to subtract buildings? :)

Reply:
It could also be anticipating more than 2,147,483,648 buildings. :-)

Actually, I'm in favor of careful attention to types. If you mean to explicitly exclude negatives, I'm all for it.

Reply:
It's impossible for an Imperator province to have less than 0 buildings, so I prefer unsigned.